frontend/src/console/components/RedeploymentToolbar/RedeploymentToolbar.tsx

```diff
@@ -33,11 +37,13 @@ export const RedeploymentToolbar: React.FC<Props> = ({
   let undeployedChanges: number = 0;
   let deploymentStatus: DeploymentStatus | null = null;
   let deploymentType: DeploymentType | null = null;
+  let isEmailVerified: boolean = false;
```

Comment:
This should default to true, otherwise all users will see this message flash while the details are being loaded from the API. That is not a good experience and can be confusing.

Reply:
Thanks, yeah it makes sense. I have changed it :+1: